.env

```diff
@@ -9,3 +9,4 @@ LND_URL=https://127.0.0.1:8081
 
 LN_ADDRESS_USER=tip
 THANKYOU="Thank you! You're awesome! <3"
+HTTPS=true
```

Comment:
What do you say we do it the other way around + call it something more descriptive.

Something like `FORCE_HTTP=false`?

Reply:
And then I do not consider that I have to put as an environment variable something that is rather internal to the program